hs_core/templates/pages/my-groups.html

```diff
@@ -140,12 +151,50 @@ <h2 class="page-title">My Groups</h2>
                                             <small class="text-muted">{{ group.gaccess.members|length }} Member{% if group.gaccess.members|length > 1 %}s{% endif %}</small>
                                         </td>
 
-                                        <td><span>{% if group.is_group_owner %}Owner{% else %}Member{% endif %}</span></td>
+                                        <td style="text-align: right"><span>{% if group.is_group_owner %}Owner{% else %}Member{% endif %}</span></td>
                                     </tr>
 
                                 {% endfor %}
                             </table>
-                        {% else %}
+                        {% endif %}
+
+                        {% if inactive_groups %}
+                            <hr>
+                            <table class="groups-table collapse" id="deleted-groups">
+                                {% for group in inactive_groups %}
+                                    <tr>
+                                        <td>
+                                            {% if group.gaccess.picture and group.gaccess.picture.url %}
+                                                <div class="group-image-wrapper">
+                                                    <div class="group-image-small"
+                                                         style="background-image: url({{ group.gaccess.picture.url }})"></div>
+                                                </div>
+                                            {% else %}
+                                                <div class="group-image-wrapper">
+                                                    <div class="group-image-small"
+                                                         style="background-image: url({{ STATIC_URL }}img/home-page/step4.png)"></div>
+                                                </div>
+                                            {% endif %}
+                                        </td>
+
+                                        <td>
+                                            <h4><a href="/group/{{ group.id }}" style="color: red">{{ group.name }}</a></h4>
```

Comment:
Have shown it to @dtarb . Without the red color there would be no way to know which ones at inactive groups. I guess there are other ways to differential though.